cirq-google/cirq_google/calibration/phased_fsim_test.py

```diff
@@ -874,3 +875,54 @@ def test_options_phase_corrected_override():
         ).zeta_chi_gamma_correction_override()
         == PhasedFSimCharacterization()
     )
+
+
+def test_try_convert_gate_to_fsim():
+    def _check(gate: cirq.Gate, expected: PhaseCalibratedFSimGate):
```

Comment:
We typically don't underscore inner functions

Reply:
Done.